app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
```

Comment:
Have changed this so it is an exact copy of the summary document viewer. 

Unless I am mistaken or have missed some JS in the output app  this will still need for the date range to be searched before being exported?